README.md

```diff
@@ -278,9 +278,11 @@ feature is currently disabled by default. To enable this feature add the followi
 ```yaml
 external-sources:
   enable: true
+  abort-after: 10m
   maven:
-    search-upstream-by-sha1: true
+    search-upstream: true
     base-url: https://search.maven.org/solrsearch/select
+    abort-after: 5m #override the global config
```

Comment:
cc @wagoodman - I know he's pretty sensitive to duplicate fields that override each other so I'd like him to chime in on where he sees this going or what his preference would be